reladomo/src/test-util/java/com/gs/fw/common/mithra/test/MithraTestResource.java

```diff
@@ -107,8 +107,8 @@
     private static final Map<MithraTestConnectionManager, List<TestDatabaseConfiguration>> configuredDatabasesPerConnectionManager =
             new HashMap<MithraTestConnectionManager, List<TestDatabaseConfiguration>>();
 
-    private final Map<MithraTestConnectionManager, List<MithraDatabaseObject>> databaseObjectPerConnectionManager =
-            new HashMap<MithraTestConnectionManager, List<MithraDatabaseObject>>(3);
+    private final Map<Object, List<MithraDatabaseObject>> databaseObjectPerConnectionManager =

```

Comment:
why?

Reply:
It used connectionManager as a key. connnectionManager is an object:

                Object connectionManager = runtimeConfig.getConnectionManager();
                databaseObjects.addAll(databaseObjectList);
                List<MithraDatabaseObject> savedDatabaseObjects = databaseObjectPerConnectionManager.get(connectionManager);
                if (savedDatabaseObjects == null)
                {
                    databaseObjectPerConnectionManager.put(connectionManager, databaseObjects);
                }

Reply:
In the context of a test, `runtimeConfig.getConnectionManager()` should always return something that implements `MithraTestConnectionManager`.  Removing the cast (below) and the type in the map will eventually break other things that expect a `MithraTestConnectionManager`.